### PR TITLE
Lpd 36494 playwright ci

### PR DESCRIPTION
**Pull request by thiago-buarqque**
Just running CI, don't mind me :sweat_smile:

### CODE REVIEW

**Comment by thiago-buarqque** [comment]
ci:help

**Comment by thiago-buarqque** [comment]
ci:test:platform-experience

File: modules/test/playwright/tests/client-extension-web/env/set_up.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+CURRENT_DIR_NAME=$(dirname ${BASH_SOURCE[0]})
+
+echo CURRENT_DIR_NAME=${CURRENT_DIR_NAME}
+
+source ${CURRENT_DIR_NAME}/../../../env/common.sh
+
+main "${@}"
```
Comment (kenjiheigel) [inline modules/test/playwright/tests/client-extension-web/env/set_up.sh:9]: Already runs `main` in line 20.

Comment (thiago-buarqque) [reply modules/test/playwright/tests/client-extension-web/env/set_up.sh:9]: @kenjiheigel  That's intentional, I am running the main function from the common file. That one declares the variable `_PORTAL_PROJECT_DIR` so I can use in line 15

Comment (thiago-buarqque) [reply modules/test/playwright/tests/client-extension-web/env/set_up.sh:9]: I could do like this, but I wanted to keep the pattern of having a main function like in other setup files

```
#!/bin/bash

CURRENT_DIR_NAME=$(dirname ${BASH_SOURCE[0]})

echo CURRENT_DIR_NAME=${CURRENT_DIR_NAME}

source ${CURRENT_DIR_NAME}/../../../env/common.sh

main "${@}"

combine_properties_files \
  $(get_tomcat_portal_ext_properties_file) \
  \
  ${_PORTAL_PROJECT_DIR}/workspaces/liferay-sample-workspace/configs/local/portal-ext.properties

default_set_up
```

**Comment by kenjiheigel** [comment]
Testing this with additional changes here: 
https://github.com/kenjiheigel/liferay-portal/pull/1762

https://github.com/kenjiheigel/liferay-portal/compare/5559d700337f203a54eeaad121f516b4eb233086^...60bd52275800792889bca7b5476abd5684a76027

**Comment by thiago-buarqque** [comment]
See https://github.com/kenjiheigel/liferay-portal/pull/1762

**Pull request closed without merging**